Puppet/Puppet.xcodeproj/project.pbxproj

```diff
@@ -792,6 +793,7 @@
 				INFOPLIST_FILE = Puppet/Info.plist;
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				OTHER_LDFLAGS = "-all_load";
```

Comment:
Thanks for the article. 4th solution seems reasonable, so if everybody supports it, i will make necessary changes.